src/vs/code/electron-main/app.ts

```diff
@@ -296,6 +296,16 @@ export class CodeApplication extends Disposable {
 
 		ipc.on('vscode:reloadWindow', (event: IpcMainEvent) => event.sender.reload());
 
+		ipc.on('vscode:webview.setIgnoreMenuShortcuts', (_event: IpcMainEvent, webContentsId: number, enabled: boolean) => {
```

Comment:
Instead of using raw IPC I would prefer to go through `IElectronService` and friends if possible. We need to move off the direct IPC usage.

Reply:
Thanks for the suggestion. Just pushed an update that moves to use channels instead. I've never worked with these directly before so let me know I've I'm using them incorrectly